### ISSUE TITLE
Add new words to internal dictionary from the app

### ISSUE THREAD

**Issue by esrk** [florisboard/florisboard#885]
Please add an option within the smart bar to add a new word to the internal/system dictionary. Few possible designs for this :

1) Long pressing on the word from suggestion list in smart bar. _Note_: This would require the unknown word to also be in the suggestion list (maybe the first option). Another usecase of this method would be, the user can also delete any words (#737).

2) Option to auto add the word to dictionary if the user types a space after the word. This is how most keyboards manage their dictionaries, hence it would be a seamless experience for new users.

**Comment by pokegamer5547** [comment]
I agree, also having the ability to delete specific word suggestions by long pressing on it(like gboard for example) would be great.
